src/components/table-hoc/TableHOC.tsx

```diff
@@ -0,0 +1,49 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import {ReduxConnect} from '../../utils/ReduxUtils';
+import {ActionBarConnected} from '../actions/ActionBarConnected';
+
+export interface ITableHOCOwnProps {
+    id: string;
+    hasActionButtons?: boolean;
+    data: any[];
+    renderData: (data: any) => React.ReactNode;

```

Comment:
Since the `data` prop has `any[]` type, I feel like the `data` parameter in `renderData` should have the same type.